Plugin/BlankCanvas/gradle.properties

```diff
@@ -0,0 +1,20 @@
+// TODO: Replace with your plugin name
+name=BlankCanvas
```

Comment:
Maybe use the [blossom gradle plugin](https://github.com/MiserableNinja/Blossom) to place these in the plugin annotation similar to what Nucleus does [here](https://github.com/NucleusPowered/Nucleus/blob/sponge-api/7/src/main/java/io/github/nucleuspowered/nucleus/PluginInfo.java). I feel it is something some people might find useful.